content/en/design/guides/designing-interviews.md

```diff
@@ -73,6 +75,60 @@ At the end of the interview, you wrap up in a way that makes participants feel a
 
 Things you should do include:
```

Comment:
Might we missing the sub title "End" here?

Reply:
It's still there.

Reply:
Alright, I have seen it